### DISCUSSION TITLE
How do I join the CAB?

### DISCUSSION THREAD

**Discussion by kozo2** [BioconductorBoards/CABWorkingGroups#16] [Q&A]
From https://www.youtube.com/watch?v=LTT1i_ACpxc

**Comment by kozo2** [comment]
There is an annual expression of interest process inviting new members to apply to join the CAB. This is announced on the Bioconductor support site and relevant Bioconductor community slack channel. Current CAB members vote on the applications received to select new members. The board is keen to have representatives from diverse backgrounds (i.e. different geographical and cultural backgrounds, genders, statistical analysis interests, career stages and experience levels).

**Comment by lshep** [comment]
Also:   Community members interested but not elected to CAB, or that would like more specialized work would be encouraged to volunteer to be part of a working group organized by the CAB/TAB --  there is a soon to be released page where we will list the active and inactive (needing someone to lead) working groups and for community members to suggest working groups.
